l2geth/rollup/config.go

```diff
@@ -14,8 +14,6 @@ type Config struct {
 	IsVerifier bool
 	// Enable the sync service
 	Eth1SyncServiceEnable bool
-	// Ensure that the correct layer 1 chain is being connected to
-	Eth1ChainId uint64
```

Comment:
We weren't using this anywhere besides in a single log statement

Reply:
I'm fine with deleting it now, it was originally used to check that we were connected to the correct chain